frontend/src/scenes/session-recordings/player/sessionRecordingPlayerLogic.ts

```diff
@@ -582,32 +611,6 @@ export const sessionRecordingPlayerLogic = kea<sessionRecordingPlayerLogicType>(
                 cancelAnimationFrame(cache.timer)
             }
         },
-        handleKeyDown: ({ event }) => {
```

Comment:
This wasn't working for some reason. Update it to use the more standard hooks variant